pvm.py

```diff
@@ -196,7 +196,9 @@ def main(RECEIVE_PORT):
 	global PORT
 	global _logger
 	p = argparse.ArgumentParser()
-	p.add_argument('--port', type=int, nargs='?', default=8001, help='The port that pvm.py will use to receive control messages. Default port is 8001')
+	p.add_argument('--port', type=int, nargs='?', default=8001, help='The port that pvm.py will use to receive control messages. '
+																		'Using the default port 8001 to control the first display. '
+																		'Using ny other port to control the second display.')
```

Comment:
```suggestion
																		'Use the default port 8001 to control the first display. '
																		'Use any other port to control the second display.')
```

@KaneBetter you sure about these massive indents?

Reply:
Indentation fixed.